tooling/src/main/java/org/opencds/cqf/tooling/acceleratorkit/Processor.java

```diff
@@ -3829,6 +3832,27 @@ public void writeDataElements(String scope, String scopePath, String context) {
         }
     }
 
+    private String retrieveLibraryShortName(){
+        switch (commonLibraryName) {
+            case "NACHCCommon":
+                return "NC";
+            case "WHOCommon":
+                return "WC";
+            case "SDHCommon":
+                return "SC";
+            default:
+                return "";
+        }
+    }
+    private ArrayList<String> retrieveProfileLibrary() {
+        for(String profileKey: this.profilesByParentProfile.keySet()){
+            if(profileKey.contains("qicore")){
+                return new ArrayList<String>(Arrays.asList("using QICore version '4.1.1'", "include QICoreCommon called QC"));
```

Comment:
Check the conventions, but I believe the convention is to remove the type specifier in this scenario ("<>" vs. "<String>").